<ui version="4.0" >
 <class>form_DebugMessages</class>
 <widget class="QDialog" name="form_DebugMessages" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DebugMessages</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTextEdit" name="textEdit" />
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cmd_connectionDump" >
       <property name="text" >
        <string>ConnectionDump</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_clear" >
       <property name="text" >
        <string>&amp;Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_close" >
       <property name="text" >
        <string>C&amp;lose</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmd_close</sender>
   <signal>clicked()</signal>
   <receiver>form_DebugMessages</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>296</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
